### PR TITLE
Add additional stats

### LINKED ISSUE #138: Add additional general statistics

### PR DESCRIPTION
**Pull request by nws321**
This pull request simply adds one more general statistic which is number of people eligible for schemes. Closes #138

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/StatisticsCommand.java [hunk 1]
```diff
@@ -23,12 +24,13 @@ public class StatisticsCommand extends Command {
...
-    public static final String MESSAGE_DISPLAY_HIGH_PRIORITY = "# Of HIGH Priority People: %s";
-    public static final String MESSAGE_DISPLAY_MEDIUM_PRIORITY = "# Of MEDIUM Priority People: %s";
-    public static final String MESSAGE_DISPLAY_LOW_PRIORITY = "# Of LOW Priority People: %s";
-    public static final String MESSAGE_DISPLAY_LOW_INCOME = "# Of People Income <= 800: %s";
-    public static final String MESSAGE_DISPLAY_APPOINTMENTS_SOON = "# Of Appointments In Next 7 Days: %s";
+    public static final String MESSAGE_DISPLAY_TOTAL_PEOPLE = "Total Number Of People: %s";
+    public static final String MESSAGE_DISPLAY_HIGH_PRIORITY = "Number Of HIGH Priority People: %s";
+    public static final String MESSAGE_DISPLAY_MEDIUM_PRIORITY = "Number Of MEDIUM Priority People: %s";
+    public static final String MESSAGE_DISPLAY_LOW_PRIORITY = "Number Of LOW Priority People: %s";
+    public static final String MESSAGE_DISPLAY_LOW_INCOME = "Number Of People Household Income <= 4000: %s";
```
Comment (arjun2598) [inline src/main/java/seedu/address/logic/commands/StatisticsCommand.java]: Just a minor grammatical suggestion - perhaps could be phrased as "Number of People with Household Income ..."

**Review by arjun2598 — COMMENTED** [review]
LGTM. Just a minor suggestion to add.

File: src/main/java/seedu/address/logic/commands/StatisticsCommand.java (hunk 1 above)
Comment (nws321) [reply src/main/java/seedu/address/logic/commands/StatisticsCommand.java]: thnks have made the change. btw is there any other general stats that u think would be useful to include as well?

**Review by Gra7ityIC3 — APPROVED** [review]
LGTM!

**Pull request merged**